### PR TITLE
Bolt demo downloadable

### PR DESCRIPTION
**Pull request by benitoThree**

### CODE REVIEW

File: sentiment_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    "### **1. Choosing and preprocessing the dataset**\n",
+    "At our webinar on April 6th, we showed how BOLT reached state-of-the-art accuracy on the [Yelp Reviews](https://github.com/huggingface/datasets/blob/master/datasets/yelp_polarity/yelp_polarity.py) dataset and demonstrated that a model trained on the [Amazon Polarity](https://huggingface.co/datasets/amazon_polarity) dataset can be used for interactive, real-time sentiment analysis. Now, we want to give you a chance to try out BOLT with a dataset of your choice. \n",
+    "\n",
+    "We provided a utility function that converts text datasets into input vectors and saves them in SVM format. The text dataset must be a CSV file where each row follows this format:\n",
+    "\n",
+    "\\<pos or neg\\>,\\<text\\> \n",
+    "\n",
+    "For example, we can have a training corpus called example_train.csv that contains the following:\n",
+    "```\n",
+    "pos,Had a great time at the webinar.\n",
```
Comment (JoshEngels) [inline sentiment_analysis.ipynb:37]: Can you say something about how many training and testing examples we need? From this I would think 2 was enough

File: text.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from thirdai import bolt, dataset
```
Comment (JoshEngels) [inline text.py:1]: This file should be in the docker container, not here

Comment (JoshEngels) [reply text.py:1]: You can put it in the docker container and add the directory to the pythonpath, or something similar

**Review by JoshEngels — CHANGES_REQUESTED** [review]
Couple of comments that shouldn't be a huge deal but I think we should make them before merging so not approving yet

**Comment by benitoThree** [comment]
Good suggestions

File: sentiment_analysis.ipynb [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    "neg,I hate slow deep learning models.\n",
+    "```"
```
Comment (benitoThree) [inline sentiment_analysis.ipynb:39]:
```suggestion
    "```\n",
    We recommend using a training corpus with at least 500,000 training samples.
```

File: sentiment_analysis.ipynb [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    "* `learning_rate`: Float - The learning rate for gradient descent. The default value is 0.0001.\n",
+    "* `epochs`: Int - The number of training epochs (a full cycle through the dataset).\n",
+    "* `verbose` (Optional): Boolean - Set to `True` to print a progress bar, accuracy, and elapsed time for each training epoch. Set to `False` otherwise. `True` by default.\n",
+    "\n",
+    "It then returns a dictionary that contains the loss value and elapsed time for each training epoch.\n",
+    "\n",
+    "\n",
+    "**Training with sparse inference in mind**\n",
+    "\n",
+    "Call the `enable_sparse_inference()` method if you plan to use sparse inference. We recommend training the network for at least one more epoch after calling this method.\n",
```
Comment (benitoThree) [inline sentiment_analysis.ipynb:154]:
```suggestion
    "If you plan to use sparse inference, we recommend calling the `enable_sparse_inference()` method before the last training epoch for accuracy improvements. For example, if the model trains for 10 epochs, this method should be called after the 9th epoch.\n",
```

File: text.py (hunk 2 above)
Comment (benitoThree) [reply text.py:1]: I want to keep it this way for now for simplicity purposes, especially since this may change after the data pipeline gets merged

**Review by JoshEngels — APPROVED** [review]
LGTM, I still think the test.py should be in the notebook. Maybe add a cell somewhere that just says "defining a couple util methods". You can merge without if you really want.
